tests/requirements.txt

```
@@ -1,35 +1,17 @@
 archspec
 boltons >=23.0.0
 charset-normalizer
-conda-content-trust
-conda-forge::anaconda-client
-conda-forge::pytest-split
-conda-forge::pytest-xprocess
-conda-forge::xdoctest
-conda-libmamba-solver >=23.11.0
+conda-content-trust >=0.2.0
 conda-package-handling >=2.2.0
```

We need to add back CLS:

```suggestion
conda-libmamba-solver >=23.11.0
conda-package-handling >=2.2.0
```